src/core_plugins/kibana/public/assets/app_timelion.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
```

Comment:
file isn't used, should we add a timelion box to the directory?

Reply:
Do a fresh pull. It should be used. Its the icon for timelion. https://github.com/nreese/kibana/blob/home_phase_one/src/core_plugins/timelion/public/register_feature.js#L8